tests/test_update.py

```diff
@@ -210,6 +212,7 @@ def test_too_big_image_update(self, bitbake_variables, connection):
             ), output
             assert "ret_code=0" not in output.stdout, output
 
+    @pytest.mark.cross_platform
```

Comment:
I think this one should be kept too. I know it says network, but it primarily tests the success path of the rootfs and boot loader integration.

Reply:
Rename?